src/components/ArticlePage/index.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+
+import { body, comment } from './samplebody';
+import CommentBox from './CommentBox';
+
+class ArticlePage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      article: {
+        id: 1,
+        title: 'Ronaldo crowned world best',
+        category: 'Sport',
+        imageUrl: 'https://images.unsplash.com/photo-1527871252447-4ce32da643c6?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2689&q=80',
+        author: {
+          fullName: 'Habeeb Olawale',
+          avatarUrl: 'https://res.cloudinary.com/vivavalinor/image/upload/v1541446448/nql15siqfajqpsrudk7b.jpg'
+        },
+        comments: [{
+          id: 1,
+          date: '12 November, 2018',
+          body: comment,
+          author: {
+            fullName: 'Chris Akanmu',
+            avatarUrl: 'https://res.cloudinary.com/vivavalinor/image/upload/v1541446448/nql15siqfajqpsrudk7b.jpg'
+          }
+        }, {
+          id: 2,
+          date: '13 November, 2018',
+          body: comment,
+          author: {
+            fullName: 'Abiodun Abudu',
+            avatarUrl: 'https://res.cloudinary.com/vivavalinor/image/upload/v1541446448/nql15siqfajqpsrudk7b.jpg'
+          }
+        }]
+      }
+    };
+  }
+
+  render() {
+    const { article } = this.state;
+    const { author } = article;
+    return (
+      <div className="site-content">
+        <p>Header here</p>
+        <div className="article-image-container">
+          <i id="back-btn" className="fa fa-arrow-circle-o-left fa-3x" />
+          <i id="bookmark-btn" className="fa fa-bookmark fa-2x" />
+          <img src={article.imageUrl} alt="article" />
+        </div>
+        <div className="article-content">
+          <span id="article-category">{article.category}</span>
+          <span id="article-title">{article.title}</span>
+          <div className="article-main">
+            <div className="author-container">
+              <img src={author.avatarUrl} alt="author" />
+              <div>
+                <span id="author-name">{author.fullName}</span>
+                <div>
+                  <span id="author-date">11, November 2018</span>
+                  •
+                  <span id="read-time">2 min read</span>
```

Comment:
The article from the database comes with readTime, please include that in your mock and pull that in here.